Translations/de.lproj/Localizable.strings

```diff
@@ -215,20 +215,20 @@ The language used in MAS/AppleID will be used in Keka's tip strings and may not
 Note that the short string is limited to 30 characters and the long string is limited to 45 characters.
 */
 "Monthly sip of coffee" = "Monthly sip of coffee";
-"Gift Keka a sip of coffee each month" = "Gift Keka a sip of coffee each month";
-"Monthly coffee" = "Monthly coffee";
-"Gift Keka a coffee each month" = "Gift Keka a coffee each month";
-"Monthly pair of coffees" = "Monthly pair of coffees";
-"Gift Keka a couple coffees each month" = "Gift Keka a couple coffees each month";
-"Monthly breakfast" = "Monthly breakfast";
-"Gift Keka a breakfast each month" = "Gift Keka a breakfast each month";
-"Tip a coffee" = "Tip a coffee";
-"Tip Keka a coffee so it can stay awake" = "Tip Keka a coffee so it can stay awake";
-"Tip a breakfast" = "Tip a breakfast";
-"Tip Keka a breakfast so it can start fresh" = "Tip Keka a breakfast so it can start fresh";
-"Tip a lunch" = "Tip a lunch";
-"Tip Keka a lunch and feed it" = "Tip Keka a lunch and feed it";
-"Tip a dinner" = "Tip a dinner";
-"Tip Keka a dinner so it keeps rolling" = "Tip Keka a dinner so it keeps rolling";
-"Tip a bunch of food" = "Tip a bunch of food";
-"Tip Keka a new bunch of food to keep rolling" = "Tip Keka a new bunch of food to keep rolling";
+"Gift Keka a sip of coffee each month" = "Schenke Keka jeden Monat einen Schluck Kaffee";
+"Monthly coffee" = "Monatlicher Kaffee";
+"Gift Keka a coffee each month" = "Schenke Keka jeden Monat einen Kaffee";
+"Monthly pair of coffees" = "Monatlich ein Paar Kaffee";
+"Gift Keka a couple coffees each month" = "Schenke Keka monatlich ein Paar Kaffee";
+"Monthly breakfast" = "Monatliches Frühstück";
+"Gift Keka a breakfast each month" = "Schenke Keka jeden Monat ein Frühstück";
+"Tip a coffee" = "Gebe Trinkgeld für einen Kaffee";
```

Comment:
This one still is more than 30 characters. Maybe "Gebe Trinkgeld für ein Kaffee"?

Reply:
changed into "Gebe Trinkgeld für Kaffee"